docs/1.concepts/data-flow/token-transfer-flow.md

```diff
@@ -0,0 +1,91 @@
+---
+sidebar_position: 2
+sidebar_label: Token Transfer
+---
+
+# Token Transfer flow
+
+:::warning Pre-requisites
```

Comment:
Since they are all part of the same sub-menu, we do not need this warning

Reply:
removed.